analytics/config.py

```diff
@@ -19,6 +19,7 @@ class DBSettings(PydanticBaseEnvConfig):
      ssl_mode: str = Field("require", alias="DB_SSL_MODE")
      db_schema: str = Field ("app", alias="DB_SCHEMA")
      slack_bot_token: str = Field(alias="ANALYTICS_SLACK_BOT_TOKEN")
+     github_token: str = Field(alias="GH_TOKEN")
```

Comment:
Added this because we now need to reference it directly within the codebase, instead of indirectly like we did previously with the `gh` CLI

Reply:
Since we are in this file, can we rename `DBSettings` to something more accurate